batch/provision.yml

```diff
@@ -10,6 +10,9 @@
     - master_instance_type: m1.medium
     - core_instance_num: 1
     - core_instance_type: m1.medium
+    - tags:
```

Comment:
Per above, let's make this a list, so that we:
- can set more than one tag
- avoid massaging the dict into a list

```yaml
vars:
  - tags:
    - Key: Cluster
      Value: hadoop
    - Key: Deployment
      Value: "{{ instance }}"
```